curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804dc.md

```diff
@@ -125,6 +125,7 @@ assert(
         <h2>Cat Form</h2>
         <form action="https://freecatphotoapp.com/submit-cat-photo">
 --fcc-editable-region--
+          
```

Comment:
Why is this extra line being added here?

Reply:
it looks better

Reply:
I guess that is a matter of opinion.  The next step does not show an extra blank line here.  This is not typically done on other steps in the project, so why now?

Reply:
it's the space for the user to write in, considering it's the first project, showing a blank line where the user can write may cause less confusion for the user

but effectively it's not there anywhere else